cluster-autoscaler/go.mod

```diff
@@ -6,7 +6,7 @@
 
 module k8s.io/autoscaler/cluster-autoscaler
 
-go 1.13
+go 1.15
```

Comment:
this might break things - I recall there's a reason for pinning it on 1.13. Let's hold off on merging this till we confirm.

Reply:
@nilo19 the go version update is caused by GO modules sync with k/k? If so, I think we should upgrade Go version anyway, as we need to vendor a lot of packages from k/k

Reply:
yeah it mirrors the `go.mod` in k/k

Reply:
OK, thanks, let's get this in. If there're some problems, we should get it fixed before releasing CA